bin/varnishd/cache/cache_req_body.c

```diff
@@ -69,6 +69,7 @@ vrb_pull(struct req *req, ssize_t maxsize, objiterate_f *func, void *priv)
 
 	req->body_oc = HSH_Private(req->wrk);
 	AN(req->body_oc);
+	req->body_oc->flags &= ~OC_F_BUSY;
```

Comment:
see previous

Reply:
Marking as resolved, agreed, this is covered by the discussion on the previous occurrence.